Userland/Utilities/seq.cpp

```diff
@@ -46,28 +48,26 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     TRY(Core::System::pledge("stdio"));
     TRY(Core::System::unveil(nullptr, nullptr));
 
-    for (size_t i = 1; i < arguments.strings.size(); i++) {
-        if (arguments.strings[i] == "--help"sv || arguments.strings[i] == "-h") {
-            out("HELP");
-            print_usage(stdout);
-            exit(0);
-        }
-    }
+    Vector<char const*> parameters;
+
+    Core::ArgsParser args_parser;
+    args_parser.add_positional_argument(parameters, "1 to 3 parameters, interpreted as LAST, FIRST LAST, or FIRST INCREMENT LAST", "parameters");
```

Comment:
these should really be three separate positional arguments, so you don't have to use `get_double`. `ArgsParser` has optional positional arguments and doubles.

Reply:
I don't think I'm the right person to implement a parser for `Vector<double>&` right now, since I'm still learning my way around the codebase and C++. That said, I agree that this would be helpful. It might also be a good way to handle negative numbers. If nobody else takes it on, I'm open to revisiting this in the future when I have more experience.

Reply:
Fair enough!